angular-client/src/components/history/carousel.component.html

```diff
@@ -0,0 +1,21 @@
+<p-carousel [value]="runs" [numVisible]="1" [numScroll]="1" [circular]="true" [showIndicators]="false">
+  <ng-template let-run pTemplate="item">
+    <div class="runcard">
+      <div class="row">
```

Comment:
yeah we should make like a row layout component instead. Or use a grid, look at the grid documentation for MUI, i didnt exactly like their implementation of it compared to the react grid but its still something to consider